libs/navigation/src/drivers/injection-tokens/category-tree-query-depth.token.ts

```diff
@@ -0,0 +1,8 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * The maximum depth of category children that the navigation driver will query.
+ * Defaults to 3.
+ */
+export const DaffNavigationCategoryTreeQueryDepth =
```

Comment:
This is platform specific (not all platforms are graphql or even configurable). We should add this as a config property of the driver module's `.forRoot()` and use the injectable config object.

Reply:
good point, although I left the token the same and pulled the value off the config instead of injecting the entire object

Reply:
changed

Reply:
LGTM